tox.ini

```diff
@@ -27,7 +27,6 @@ setenv =
   VIRTUAL_ENV={envdir}
   QISKIT_SUPPRESS_PACKAGING_WARNINGS=Y
 deps =
-  git+https://github.com/Qiskit/qiskit-ibmq-provider
```

Comment:
We can drop it from the main test environment above as well, right?

Reply:
Yes, thanks for catching it. The ibmq-provider mentions in the documentation will be dropped in the docs PR so I won't remove them here.